### DISCUSSION TITLE
Application suddenly crashed after running trouble free for months

### DISCUSSION THREAD

**Discussion by siddhsql** [sveltejs/kit#12825] [Q&A]
I have a SvelteKit application that was running smoothly for many weeks. Suddenly it crashed yesterday with this message:

```
node:events:497
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/home/xxx/xxx/.svelte-kit/output/client/_app/immutable/chunks/entry.D0G_PEZa.js'
Emitted 'error' event on ReadStream instance at:
    at emitErrorNT (node:internal/streams/destroy:169:8)
    at emitErrorCloseNT (node:internal/streams/destroy:128:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/xxx/xxx/.svelte-kit/output/client/_app/immutable/chunks/entry.D0G_PEZa.js'
}

Node.js v20.14.0
```

when I did a `ls`, yes I don't see any such file in the directory:

```
$ ls .svelte-kit/output/client/_app/immutable/chunks
Button.C19rDXsl.js  entry.JAmeqamq.js  index.nxLW6K-w.js  index.r5IGbbBn.js  scheduler.CdpBKkh_.js
```

however, the app was working fine without that file for many weeks and I did not access a page that I hadn't accessed before. 

Can someone explain this?

**Comment by brunnerh** [comment]
Getting such errors is expected when updating the application and a user still has the old version open. Paths will still reference the old files but the application should do a full reload after such errors are encountered.

If the errors are _not_ temporary that suggests to me an incomplete update of the application, leading to broken file references.
